frontend/app/src/views/source-data/TaskManagement.vue

```diff
@@ -319,132 +291,132 @@ export default {
-          <v-checkbox readonly :model-value="item.raw.loadTypeFacility" />
-        </template>
-        <template #item.loadTypeHds="{ item }">
-          <v-checkbox readonly :model-value="item.raw.loadTypeHds" />
-        </template>
-        <template #item.loadTypeOrg="{ item }">
-          <v-checkbox readonly :model-value="item.raw.loadTypeOrg" />
+          {{ $filters.formatDate(item.raw.fileExtractedDate) }}
         </template>
-        <template #item.loadTypeOFRelationship="{ item }">
-          <v-checkbox readonly :model-value="item.raw.loadTypeOFRelationship" />
-        </template>
-        <template #item.loadTypeOORelationship="{ item }">
-          <v-checkbox readonly :model-value="item.raw.loadTypeOORelationship" />
-        </template>
-        <template #item.loadTypeIORelationship="{ item }">
-          <v-checkbox readonly :model-value="item.raw.loadTypeIORelationship" />
-        </template>
-        <template #item.loadTypeWOXref="{ item }">
-          <v-checkbox readonly :model-value="item.raw.loadTypeWOXref" />
-        </template>
-        <template #item.loadTypeWPIXref="{ item }">
-          <v-checkbox readonly :model-value="item.raw.loadTypeWPIXref" />
+        <template #item.runTypes="{ item }">
+          <v-chip v-if="item.raw.loadTypeFacility" variant="elevated">
+            {{ RunTypes.loadTypeFacility }}
+          </v-chip>
+          <v-chip v-if="item.raw.loadTypeHds" variant="elevated">
+            {{ RunTypes.loadTypeHds }}
+          </v-chip>
+          <v-chip v-if="item.raw.loadTypeOrg" variant="elevated">
+            {{ RunTypes.loadTypeOrg }}
+          </v-chip>
+          <v-chip v-if="item.raw.loadTypeOFRelationship" variant="elevated">
+            {{ RunTypes.loadTypeOFRelationship }}
+          </v-chip>
+          <v-chip v-if="item.raw.loadTypeOORelationship" variant="elevated">
+            {{ RunTypes.loadTypeOORelationship }}
+          </v-chip>
+          <v-chip v-if="item.raw.loadTypeIORelationship" variant="elevated">
+            {{ RunTypes.loadTypeIORelationship }}
+          </v-chip>
+          <v-chip v-if="item.raw.loadTypeWOXref" variant="elevated">
+            {{ RunTypes.loadTypeWOXref }}
+          </v-chip>
+          <v-chip v-if="item.raw.loadTypeWPIXref" variant="elevated">
+            {{ RunTypes.loadTypeWPIXref }}
+          </v-chip>
         </template>
         <template #item.actions="{ item }">
-          <v-tooltip location="bottom">
-            <template #activator="{ props }">
-              <v-icon
-                v-bind="props"
-                size="small"
-                class="me-2"
-                label="VIEW"
-                @click="redirectToView(item.key)"
-              >
-                mdi-format-list-bulleted
-              </v-icon>
-            </template>
-            <span>View source data</span>
-          </v-tooltip>
+          <slot v-if="item.raw.statusCode === 'UPLOAD_IN_PROGRESS'">
```

Comment:
not sure how useful would be to consider constants for the status codes?

Reply:
Yes For Sure